src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import {
+  createStore, combineReducers, applyMiddleware,
+} from 'redux';
+import logger from 'redux-logger';
+
+import bookReducer from './books/books';
+
+const reducer = combineReducers({
+  bookReducer,
+});
```

Comment:
- [ ] Kindly have to add your `categories reducer` here also, I noted that you had added your categories Reducer before but after that, you have deleted it, Kindly write it back because it's a requirement.

You have deleted it in this [commit](https://github.com/Mutalenic/Bookstore/pull/2/commits/c7e9ef1df6c8208cb764260aebf87192dc78218d)

The requirements 
![image](https://user-images.githubusercontent.com/35535130/163200131-e102168c-b8bb-4855-9ef3-7dfb7cf21328.png)